website/docs/r/identitystore_group_membership.html.markdown

```diff
@@ -0,0 +1,65 @@
+---
+subcategory: "IdentityStore"
+layout: "aws"
+page_title: "AWS: aws_identitystore_group_membership"
+description: |-
+Terraform resource for managing an AWS IdentityStore Group Membership.
+---
+
+# Resource: aws_identitystore_group_membership
+
+Terraform resource for managing an AWS IdentityStore Group Membership.
+
+## Example Usage
```

Comment:
Could you rename the `test` to `example` to be in line with the other identitystore docs?